osgeo_importer/models.py

```diff
@@ -150,22 +150,20 @@ def filesize(self):
         """
         Humanizes the upload file size.
         """
-        if not self.size:
-            uploaded_file = self.uploadfile_set.first()
-
-            if uploaded_file:
-                self.size = uploaded_file.file.size
-                self.save()
-            else:
-                return
-
+        size = 0
+        if not self.size or self.size == 0:
+            if self.uploadfile_set.count() > 0:
+                for uf in self.uploadfile_set.all():
+                    size += uf.file.size
+        self.size = size
+        self.save()
         return sizeof_fmt(self.size)
 
     def file_url(self):
         """
         Exposes the file url.
         """
-        return self.uploadfile_set.first().file.url
+        return '' # self.uploadfile_set.first().file.url
```

Comment:
Looks like this may be used in `osgeo_importer/static/osgeo_importer/partials/upload.html` around line 108. Can we check whether this breaks the `Download` link in the UI? If so, maybe we need to just delete it, or is there some other way we should support it (e.g. multiple download links)?